security/security-core/src/main/java/io/camunda/security/configuration/InitializationConfiguration.java

```diff
@@ -19,6 +22,7 @@ public class InitializationConfiguration {
 
   private List<ConfiguredUser> users = new ArrayList<>();
   private List<ConfiguredMapping> mappings = new ArrayList<>();
+  private Map<String, Map<String, Collection<String>>> defaultRoles = new HashMap<>();
```

Comment:
❓ I wonder if this would be better named something like `defaultRoleMemberships` because when I first read `defaultRoles` I think I'm interacting with roles themselves but I'm actually only dealing with the memberships.

Reply:
Yeah I agree it's a bit confusing but it's what we've settled on in the meeting :shrug: